slick/__init__.py

```diff
@@ -44,7 +44,7 @@ def add_widget(self, widget):
 
 # Setup the main Flask app
 app = Slick(__name__)
-app.config.from_envvar('SLICK_CONFIG_FILE')
```

Comment:
The config for this project is almost superfluous, and having it be in a set location makes getting it running easier.

I don't think anyone is actually using multiple custom locations, but in case they are, I'll let from_envvar override from_py, but I'll do that in another issue.